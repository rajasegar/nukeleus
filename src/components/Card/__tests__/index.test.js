import React from 'react';
import { renderWithTheme } from '../../../test-utils';

import { Card } from '../index';

describe('Card component test suite', () => {
  it('should render Card', async () => {
    const { container } = renderWithTheme(<Card>Card component </Card>);
    expect(container.firstChild).toMatchSnapshot();
  });
});
