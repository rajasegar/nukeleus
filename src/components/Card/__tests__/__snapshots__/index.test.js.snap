// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component test suite should render Card 1`] = `
.c0 {
  cursor: pointer;
  padding: 20px 32px 32px;
  position: relative;
  border: 1px solid #EBEFF3;
  box-shadow: 0px 2px 4px rgba(18,52,77,0.06);
  border-radius: 12px;
  width: 290px;
  margin-bottom: 24px;
  margin-right: 24px;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  opacity: 1;
  pointer-events: all;
}

.c0:hover {
  box-shadow: 0px 2px 16px rgba(18,52,77,0.16);
}

<div
  class="c0"
>
  Card component 
</div>
`;
